jetty-websocket/jetty-websocket-common/src/test/java/org/eclipse/jetty/websocket/common/OutgoingMessageCapture.java

```diff
@@ -295,6 +295,18 @@ public long getMaxTextMessageSize()
                 return maxMessageSize;
             }
 
+            @Override
+            public long getMaxFrameSize()
+            {
+                return 0;
```

Comment:
What does a maxFrameSize of 0 mean?

Reply:
It just means we don't enforce the maxFrameSize. But here this is in the method `getFakeSession()` so this is probably not even used here.